dashboard/app/public_json_api.go

```diff
@@ -66,9 +74,11 @@ func getExtAPIDescrForBugPage(bugPage *uiBugPage) *publicAPIBugDescription {
 			}
 			commit := bugPage.BisectCause.Commit
 			return &vcsCommit{
-				Title: commit.Title,
-				Hash:  commit.Hash,
-				Link:  commit.Link,
+				Title:  commit.Title,
+				Hash:   commit.Hash,
+				Link:   commit.Link,
+				Repo:   repoName,
+				Branch: branchName,
```

Comment:
First crash repo does not necessary match bisection repo.

Reply:
Hm.. I'll use 
https://github.com/google/syzkaller/blob/ca69c785c2ed2872a7c1a427fc446a9d363a984f/syz-ci/jobs.go#L397 

then.

Reply:
It is better to store Repo in db and don't worry about the config changes, but as a first version mgr.mgrcfg.Repo should be fine.

Reply:
Thanks. It is already in DB.